bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/handler/AbstractHomeConnectThingHandler.java

```diff
@@ -541,9 +541,11 @@ protected void updateChannel(ChannelUID channelUID) {
             } catch (CommunicationException e) {
                 logger.debug("API communication problem while trying to update! thing={}, haId={}, error={}",
                         getThingLabel(), getThingHaId(), e.getMessage());
+                updateState(channelUID, UnDefType.UNDEF);
             } catch (AuthorizationException e) {
                 logger.debug("Authentication problem while trying to update! thing={}, haId={}", getThingLabel(),
                         getThingHaId(), e);
+                updateState(channelUID, UnDefType.UNDEF);
```

Comment:
Shouldn't you set the Thing status to OFFLINE here instead?

Reply:
`handleAuthenticationError` is called in this case which will reinitialize the bridge thing and all its related things status.

Reply:
But then I wouldn't set the state to UNDEF, but simply leave the one that was set before.
UNDEF essentially means it has a state, which cannot be properly mapped to openHAB...

Reply:
Ok, reverted.